plugins/scaffolder/src/components/fields/RepoUrlPicker/utils.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export function serializeFormData(data: {
```

Comment:
i know that this naming kinda makes sense within this particular folder since it's local, but just wondered if they might be named `parseRepoPickerUrl` and `stringifyRepoPickerUrl` which is more explicit and consistent with other such pairs, and possibly they could even be exported in some future for others' benefit